packages/validtion-errors/src/ValidationErrors.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import type * as shared from '@contentful/field-editor-shared';
+import { entityHelpers } from '@contentful/field-editor-shared';
+import { TextLink, List } from '@contentful/forma-36-react-components';
+
+import * as styles from './styles';
+
+export type ValidationError = {
+  name: string;
+  message: string;
+  conflicting?: Array<{ sys: { id: string } }>;
+};
+
+type UniquenessErrorProps = {
+  error: ValidationError;
+  space: shared.SpaceAPI;
+  localeCode: string;
+  defaultLocaleCode: string;
+  getEntryURL: (entry: shared.Entry) => string;
+};
+
+function UniquenessError(props: UniquenessErrorProps) {
+  const [state, setState] = React.useState<{
+    loading: boolean;
+    entries: { title: string; href: string }[];
+  }>({
+    loading: true,
+    entries: [],
+  });
+
+  const contentTypesById = React.useMemo(
+    (): Record<string, shared.ContentType> =>
+      // Maps ID => Content Type
+      props.space.getCachedContentTypes().reduce(
+        (prev, ct) => ({
+          ...prev,
+          [ct.sys.id]: ct,
+        }),
+        {}
+      ),
+    [props.space]
+  );
+
+  React.useEffect(() => {
+    setState((state) => ({ ...state, loading: true }));
+
+    const getTitle = (entry: shared.Entry) =>
+      entityHelpers.getEntryTitle({
+        entry,
+        defaultTitle: 'Untitled',
+        localeCode: props.localeCode,
+        defaultLocaleCode: props.defaultLocaleCode,
+        contentType: contentTypesById[entry.sys.contentType.sys.id],
+      });
+
+    const query = {
+      'sys.id[in]': props.error.conflicting?.map((entry) => entry.sys.id).join(','),
+    };
+
+    props.space.getEntries<shared.Entry>(query).then(({ items }) => {
+      const entries = items.map((entry) => ({
+        title: getTitle(entry),
+        href: props.getEntryURL(entry),
+      }));
+
+      setState({
+        loading: false,
+        entries,
+      });
+    });
+  }, [props]);
```

Comment:
Are we sure that we want to react on all props changes here?

Reply:
Realistically we want to only refetch when the error changes but I also don't want to lie to React about the dependencies since it may cause some tricky bugs in end-users code.

What we can do is comparing the `error.conflicting` ids with those we have in `state.entries` and only fetch different, how about that @suevalov ?

Reply:
sounds good

Reply:
Done. Let me know what you think now